docs/03-tutorials/00-serverless/svls-11-set-asynchronous-connection-of-functions.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Set asynchronous communication between Functions
+---
+
+This tutorial shows how you can connect two Functions using asynchronous communication. You can do it thanks to the In-cluster Eventing embedded in Kyma. The tutorial is based on Function from [Incluster Eventing example](https://github.com/kyma-project/examples/pull/188).
+
+## Prerequisites
+
+Before you start, make sure you have these tools installed:
+
+- [Kyma CLI](https://github.com/kyma-project/cli)
+- Kyma installed locally or on a cluster
+
+## Steps
+
+1. Export the `KUBECONFIG` variable:
+   ```bash
+   export KUBECONGIG={KUBECONFIG_PATH}
+   ```
+2. Create two folders in your project - `emitter` and `receiver`.
+3. Go to your `emitter` folder and run the `init` Kyma CLI command to initiate your Function:
+
+   ```bash
+   kyma init function
+    ```
+
+  The `init` command creates these files in your workspace folder:
+
+  - `config.yaml`	with the Function's configuration
+
+>**NOTE:** See the detailed description of all fields available in the [`config.yaml` file](../../05-technical-reference/svls-06-function-configuration-file.md).
+
+  - `handler.js` with the Function's code and the simple "Hello Serverless" logic
+  
+  - `package.json` with the Function's dependencies
+
+4. In the `config.yaml` file configure your API Rule to allow for the incoming outside traffic (using HTTP). Enter your cluster domain in the host key:
+
+  ```yaml
+  apiRules:
+    - name: incomming-http-trigger
+      service:
+        host: incomming.inclusterevent.acme.shoot.canary.k8s-hana.ondemand.com
+      rules:
+        - methods:
+            - GET
+          accessStrategies:
+            - handler: allow
+  ```
+
+5. Provide your Function logic in the `handler.js` file. In the example you can find a simple code sanitization:
+
+   ```js
+   const { v4: uuidv4 } = require('uuid');
+   module.exports - {
+       main: function (event, context) {
+           let sanitsedData = sanitise(event.data)
+           var eventOut=event.buildResponseCloudEvent(uuidv4(), "sap.kyma.custom.acme.payload.sanitised.v1", sanitisedData);
+           eventOut.source="kyma"
+           eventOut.specversion="1.0"
+           event.publishCloudEvent(eventOut);
+           console.log(`Payload pushed to sap.kyma.custom.acme.payload.sanitised.v1`,eventOut)
+           return eventOut;
+       }
+   }
+   let sanitise = (data)=>{
+       console.log(`sanitising data...`)
+       console.log(data)
+       return data
+   }
+   ```
+
```

Comment:
add some extra guidance:

Ie:

Please note the `sap.kyma.custom.acme.payload.sanitised.v1`. This is the event type the emitter is publishing to. The one used here is an  example of-course and you can choose a  different one that better describes your usecase.  Keep in mind the following constraints (pls validate with @k15r ):
 -  it must consist of seven elements separated by dots . 
 -   First three elements must remain unchanged (`sap.kyma.custom`).
 -  The last element specifies the version of the event.

The receiving function must subscribe to this event type. 

Also...

Please note that `event` object provides convinience functions to build and publish events (the content of point 7 bellongs here)